src/client/Microsoft.Identity.Client/MsalError.cs

```diff
@@ -1030,5 +1031,14 @@ public static class MsalError
         /// <para>Mitigation</para> Use WithTenantId at the request level instead.
         /// </summary>
         public const string RegionalAndAuthorityOverride = "authority_override_regional";
+
+        /// <summary>
+        /// <para>What happens?</para>The token cache does not contain a token with an OBO cache key that
+        /// matches the <c>longRunningProcessSessionKey</c> passed into <see cref="ILongRunningWebApi.AcquireTokenInLongRunningProcess"/>.
+        /// <para>Mitigation</para>Call <see cref="ILongRunningWebApi.InitiateLongRunningProcessInWebApi"/> with a new <c>longRunningProcessSessionKey</c>
```

Comment:
actually if it does not exist the cache, how can we call with a key that does not exist? Is it the right sentence?

Reply:
Rephrased a little.

>         /// <para>What happens?</para>The token cache does not contain a token with an OBO cache key that
>         /// matches the <c>longRunningProcessSessionKey</c> passed into <see cref="ILongRunningWebApi.AcquireTokenInLongRunningProcess"/>.
>         /// <para>Mitigation</para> Call <see cref="ILongRunningWebApi.InitiateLongRunningProcessInWebApi"/> with this <c>longRunningProcessSessionKey</c> 
>         /// first or call <see cref="ILongRunningWebApi.AcquireTokenInLongRunningProcess"/> with an already used <c>longRunningProcessSessionKey</c>.